tests/localization_fixture/deno.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lock": false,
+  "tasks": {
+    "start": "deno run -A --watch=static/,routes/ dev.ts",
+    "update": "deno run -A -r https://fresh.deno.dev/update ."
+  },
+  "imports": {
+    "$fresh/": "https://raw.githubusercontent.com/deer/fresh/feat_plugin_islands/",
```

Comment:
I hope to remove all this nonsense once https://github.com/denoland/fresh/pull/1472 gets merged. Then we can just use the standard version.